tasks/install_package_redhat.yml

```diff
@@ -0,0 +1,12 @@
+---
+- name: add Hashicorp repo
+  command: yum-config-manager --add-repo=yum-config-manager --add-repo https://rpm.releases.hashicorp.com/rhel/hashicorp.repo
+  args:
+    creates: /etc/yum.repos.d/hashicorp.repo
+  become: true
+
+- name: install Boundary
+  rpm:
```

Comment:
Please use `package` or `yum` here, **rpm** is not an ansible module.

Reply:
fixed